labs/a3.md

```diff
@@ -78,14 +78,24 @@ sudo apt install --no-install-recommends qemu-kvm qemu-utils libvirt-bin virtins
 
 ### acquire installation media
```

Comment:
Nit: the capitalization is inconsistent between labs, should we standardize it?

Reply:
Good point, I'd rather have all the lab sections properly capitalized; I'll go back and change them.